Doc/library/traceback.rst

```diff
@@ -653,5 +671,86 @@ This last example demonstrates the final few formatting functions:
    ['  File "spam.py", line 3, in <module>\n    spam.eggs()\n',
     '  File "eggs.py", line 42, in eggs\n    return "bacon"\n']
    >>> an_error = IndexError('tuple index out of range')
-   >>> traceback.format_exception_only(type(an_error), an_error)
+   >>> traceback.format_exception_only(an_error)
    ['IndexError: tuple index out of range\n']
+
+
+Examples of Using :class:`TracebackException`
+---------------------------------------------
+
+With the helper class, we have more options::
```

Comment:
if we use the `.. doctest::` directive, this REPL example will get tested in CI, which could help ensure it stayes up to date when the traceback format is changed in the future:

```suggestion
With the helper class, we have more options:

.. doctest::
```